### PR TITLE
V4.20.1

### PR DESCRIPTION
**Pull request by yotamberk**
release for v4.20.1!!!

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,6 +1,43 @@
...
+### General
+	Added Release checklist 
+	Added collapsible items for objects in graph3d doc
+
+### Network
+	FIX #3203: Set dimensions properly of images on initialization
+	FIX #3198: Small fix on ref usage in DataGroup
+	FIX #3170: Refactoring of Node Drawing 
+	FIX #3108: Reverse nodes returned with 'from' and 'to' directions
+	FIX #3105: Adjusting timeline TimeStep.roundToMinor
```
Comment (wimrijnders) [inline HISTORY.md]: Should be under timeline

File: HISTORY.md [hunk 2]
```diff
@@ -1,6 +1,43 @@
...
+	FEAT #3154: Caching to Range getMillisecondsPerPixel function
```
Comment (wimrijnders) [inline HISTORY.md]: Should be under timeline

File: HISTORY.md (hunk 1 above)
Comment (yotamberk) [inline HISTORY.md]: I've fixed it

File: HISTORY.md [hunk 3]
```diff
@@ -1,6 +1,43 @@
...
+	FIX #1105: Fix usage of clustering with hierarchical networks
+	FIX #3133: Protect Network from zero and negative mass values
+	FIX #3163: Prevent crashes from invalid id's in `Clustering.findNode()`
+	FIX #3106: Ensure start and end of stabilization progress events is sent
+	FIX #3015: Properly handle newline escape sequences in strings for DOT
+	FIX Refactoring of LayoutEngine
+	FIX #2990: Edge labels turn bold on select and hover
+	FIX #2959: Changed order of (de)select events for network
+	FIX #3091: Added param 'direction' to Network.getConnectedNodes()
+    FIX #3085: Add prefix  to cancelAnimationFrame()
```
Comment (wimrijnders) [inline HISTORY.md]: indent

**Pull request merged**